kubernetes/storageclass.yaml.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: eosrate-storage

```

Comment:
lets call this new storage class `eden-storage`

Reply:
I created a new one called eden-storage